docs/_tutorials/getting_started.md

```diff
@@ -12,4 +12,319 @@ redirect_from:
+
+Back on your app configuration page, click on **Interactivity & Shortcuts** on the left side. You'll see that there's another **Request URL** box.
+
+By default, Bolt is configured to use the same endpoint for interactive components that it uses for events, so use the same request URL as above (in the example, it was `https://8e8ec2d7.ngrok.io/slack/events`). Press the **Save Changes** button in the lower right hand corner, and that's it. Your app is set up to handle interactivity!
+
+![Configuring a Request URL](../assets/request-url-config.png "Configuring a Request URL")
+
+Now, let's go back to your app's code and add interactivity. This will consist of two steps:
+- First, your app will send a message that contains a button.
+- Next, your app will listen to the action of a user clicking the button and respond
+
+Below, the code from the last section is modified to send a message containing a button rather than just a string:
+
+```python
+import os
+from slack_bolt import App
+
+# Initializes your app with your bot token and signing secret
+app = App(
+    token=os.environ.get("SLACK_BOT_TOKEN"),
+    signing_secret=os.environ.get("SLACK_SIGNING_SECRET")
+)
+
+# Listens to incoming messages that contain "hello"
+@app.message("hello")
+def message_hello(message, say):
+    # say() sends a message to the channel where the event was triggered
+    say(
+        blocks=[
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": f"Hey there <@{message['user']}>!"
+                },
+                "accessory": {
+                    "type": "button",
+                    "text": {
+                        "type": "plain_text",
+                        "text": "Click Me"
+                    },
+                    "action_id": "button_click"
+                }
+            }
+        ],
+        text=f"Hey there <@{message['user']}>!"
+    )
+
+# Start your app
+if __name__ == "__main__":
+    app.start(port=int(os.environ.get("PORT", 3000)))
+```
+
+The value inside of `say()` is now an object that contains an array of `blocks`. Blocks are the building components of a Slack message and can range from text to images to datepickers. In this case, your app will respond with a section block that includes a button as an accessory. Since we're using `blocks`, the `text` is a fallback for notifications and accessibility.
+
+You'll notice in the button `accessory` object, there is an `action_id`. This will act as a unique identifier for the button so your app can specify what action it wants to respond to.
+
+> 💡 The [Block Kit Builder](https://api.slack.com/tools/block-kit-builder) is an simple way to prototype your interactive messages. The builder lets you (or anyone on your team) mockup messages and generates the corresponding JSON that you can paste directly in your app.
```

Comment:
Nice to have a link to Block Kit Builder here! `https://api.slack.com/tools/block-kit-builder ` is now redirected to `https://app.slack.com/block-kit-builder`. Using `https://api.slack.com/tools/block-kit-builder` may be safer in the long run.

Reply:
Great catch! I've fixed this in commit 7d10d7f and will also update the `bolt-js` guide.

Reply:
Thanks!

>Using https://api.slack.com/tools/block-kit-builder may be safer in the long run.

I just noticed this sentence was incorrect ... but thanks for understanding this 😅